lib/modules/hosts/wikipedia.js

```diff
@@ -1,22 +1,33 @@
 import { ajax } from '../../environment';
+import { $ } from '../../vendor';
 
 export default {
 	moduleID: 'wikipedia',
 	name: 'wikipedia',
-	domains: ['wikipedia.org'],
-	attribution: false,
-	detect: ({ href }) => (/^https?:\/\/([A-z]{2}|simple)?\.(?:m\.)?wikipedia\.org\/wiki\/(.+)/i).exec(href),
+	domains: ['wikipedia.org', 'wikipedia.com'],
+	logo: 'https://en.wikipedia.org/static/favicon/wikipedia.ico',
+	detect: ({ href }) => (/^https?:\/\/([A-z]{2}|simple)?\.(?:m\.)?wikipedia\.(?:com|org)\/wiki\/([^#\?]+)/i).exec(href),
 	async handleLink(href, [, country, article]) {
 		const data = await ajax({
-			url: 'https://noembed.com/embed',
-			data: { url: `http://${country}.wikipedia.org/wiki/${article}` },
+			url: `https://${country}.wikipedia.org/w/api.php?action=parse&format=json&prop=text|displaytitle&section=0&page=${article}&origin=*`,
 			type: 'json',
+			headers: {
+				// See https://www.mediawiki.org/wiki/API:Main_page#Identifying_your_client
+				'Api-User-Agent': 'Reddit-Enhancement-Suite/5.0 ( https://github.com/honestbleeps/Reddit-Enhancement-Suite/ )',
+			},
 		});
 
+		// Clean up returned data
+		const $wikiData = $(`<div>${data.parse.text['*']}</div>`);
```

Comment:
Maybe use `$('<div>', { html: data.parse.text['*'] });` so it degrades a bit more gracefully if `data.parse.text['*']` is null or undefined or invalid html